Root.tsx

```diff
@@ -54,17 +70,31 @@ const AuthStackNavigator = () => {
 const DrawerContent = (props: DrawerContentComponentProps) => {
   const { logout, user } = useContext(AuthContext);
 
+  function updateInfoScreen(role: string) {
+    return () => {
+      if (role === Roles.PATIENT) {
+        props.navigation.navigate("UpdateInfo");
+      }
+    };
+  }
   return (
     <DrawerContentScrollView
       contentContainerStyle={styles.contentContainer}
       scrollEnabled={false}
     >
-      <DrawerItemList {...props} />
+      <View style={styles.drawerItemListContainer}>
+        <DrawerItemList {...props} />
+      </View>
       <View style={styles.logoutContainer}>
+        <TouchableOpacity onPress={updateInfoScreen(user?.role ?? "")}>
```

Comment:
In this line I think you could do this instead:

`<TouchableOpacity onPress={() => updateInfoScreen(user?.role ?? "")}>`

By doing this, you could change the `updateInfoScreen` function so it doesn't return anything (right now it's a function that returns another function).